ams/dashboard/src/utils/request.ts

```diff
@@ -22,7 +22,7 @@ import useStore from '@/store'
 import router from '@/router'
 
 // http://10.196.98.23:19111/
-export const baseURL = '/'
+export const baseURL = '/mock'
 
```

Comment:
I don't think it's necessary to change the baseUrl to `/mock` by default here, maybe add a comment line, if you need to use the mock service, you can change the baseUrl to `/mock`.